nextcloudappstore/core/models.py

```diff
@@ -20,6 +21,19 @@ def search(self, terms, lang):
         query = reduce(lambda x, y: x & y, predicates, Q())
         return queryset.filter(query)
 
+    def get_compatible(self, platform_version):
+        apps = App.objects.prefetch_related('translations', 'screenshots',
+                                            'releases', 'releases__databases',
+                                            'releases__php_extensions').all()
+
+        def app_filter(app):
+            for release in app.releases.all():
+                if release.is_compatible(platform_version):
```

Comment:
No, it needs to check all releases for a given app. This change would mean that whatever release it checks first determines whether the app as a whole is compatible with a platform version.